packages/side-nav/src/vaadin-side-nav-base-styles.js

```diff
@@ -28,6 +28,10 @@ export const sideNavItemBaseStyles = css`
     -webkit-appearance: none;
     appearance: none;
     flex: none;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    background: transparent;
```

Comment:
These styles essentially reset the `<button>` user agent defaults, so I moved them to `src` folder.